builds/psvita/Makefile

```diff
@@ -27,7 +27,7 @@ PREFIX  = arm-vita-eabi
 CC      = $(PREFIX)-gcc
 CXX      = $(PREFIX)-g++
 CFLAGS  = -Wl,-q -O3 -DPSP2 -DHAVE_LIBSPEEXDSP \
-		-DHAVE_SLOW_CPU -DSUPPORT_AUDIO -DHAVE_LIBSNDFILE \
+		-DWANT_FASTWAV=1 -DSUPPORT_AUDIO -DHAVE_LIBSNDFILE \
```

Comment:
That =1 thing was in FMMIDI because we offer 2 options (1 and 2). For your case this is not needed, you can remove the "=1"